features/google/src/model/importers/utils/lang-constants.js

```diff
@@ -0,0 +1,31 @@
+export const langConstants = {
+    AccessLogImporter: {
+        english: { regex: /\/Access Log Activity\/.*\.csv$/ },
+        german: { regex: /\/Zugriffsprotokollaktivitäten\/.*\.csv$/ },
+        danish: { regex: /\/Aktiviteter i adgangsloggen\/.*\.csv$/ },
+    },
+    ActivitiesImporter: {
+        english: { regex: /\/My Activity\/.*\.html$/ },
+        german: { regex: /\/Meine Aktivitäten\/.*\.html$/ },
+        danish: { regex: /\/Mine aktiviteter\/.*\.html$/ },
+    },
+    SemanticLocationsImporter: {
+        english: {
+            regex: /\/[^/]+\/Semantic Location History\/\d+\/[^.]+\.json$/,
+        },
+        german: {
+            regex: /\/[^/]+\/Semantic Location History\/\d+\/[^.]+\.json$/,
+        },
+        danish: {
+            regex: /\/[^/]+\/Semantic Location History\/\d+\/[^.]+\.json$/,
+        },
+    },
+};
+export function matchRegex(path, importer) {
+    const importerRegex = langConstants[importer];
+    const language = Object.keys(importerRegex).find((key) =>
+        importerRegex[key].regex.test(path)
+    );
```

Comment:
OK, so you're testing for every possible language, but then you're not actually returning the language, but the fact that it matched or not. So maybe consider using a single regex for any language? It would involve adding optional strings for new languages, but it would also mean that you don't need to try and match several times if there's a single regex (as is the case sometimes)